shared/data/movies/src/main/java/com/my/movie/dto/MovieDto.kt

```
@@ -28,7 +31,9 @@ data class MovieDto(
     private val voteAverage: Float?
 ) {
     fun toValueObject(): Movie = Movie(
+        id = id ?: UUID.randomUUID().toString().toInt(),
```

Хорошая идея - в случае отсутствия генерировать ID самостоятльено, но в реальном приложении на самом деле кмк кинуть Exception - иначе будет рассинхрон айдишников с сервером и такие баги будет сложно искать - это принцип fail fast - типа если что-то пошло не так что не позволяет корректно работать - то лучше пусть падает, но это так FYI - оч холиварный момент